tests/foreman/ui/test_adusergroup.py

```diff
@@ -323,7 +349,7 @@ def test_positive_update_external_roles(self):
         @assert: User has access to all NEW functional areas that are assigned
         to aforementioned UserGroup.
         """
-        strategy, value = locators['login.loggedin']
+        self.check_external_user()
```

Comment:
That test case is actually working and is not blocked by mentioned BZ, but is completely unstable and require huge delays (more than 10-15 minutes) between some steps, so can be analyzed in future. Maybe if initial BZ will be fixed, behavior will change in some ways